File: latihan2/php/hardware.php
<?php
    include "product.php";

    class hardware extends product{
        // atribut
        private $brand = "blank";
        private $model = "blank";
    
        // constructor
        function __construct(){}

        // Setter dan Getter

        function setBrand($brand){
            $this->brand = $brand;
        }
        function getBrand(){
            return $this->brand;
        }

        function setModel($model){
            $this->model = $model;
        }
        function getModel(){
            return $this->model;
        }
        
        // destructor
        function __destruct(){}

    }
?>

File: latihan2/python/hardware.py
from product import product

class hardware(product):
    # atribut
    __brand = "blank"
    __model = "blank"

    # constructor
    def __init__(self, model="blank", brand="blank"):
        self.__brand = brand
        self.__model = model

    # Setter dan Getter

    def setBrand(self, brand):
        self.__brand = brand
        
    def getBrand(self):
        return self.__brand 

    def setModel(self, model):
        self.__model = model
        
    def getModel(self):
        return self.__model 
